content/en/docs/languages/cpp/getting-started.md

```diff
@@ -267,23 +268,16 @@ endif()
 # set the path to the directory containing "oatpp" package configuration files
 include_directories(${OATPP_ROOT}/src)
 
-include_directories(${OPENTELEMETRY_ROOT}/api/include)
-include_directories(${OPENTELEMETRY_ROOT}/sdk/include)
-include_directories(${OPENTELEMETRY_ROOT}/sdk/src)
-include_directories(${OPENTELEMETRY_ROOT}/exporters/ostream/include)
-
-find_library(OPENTELEMETRY_COMMON_LIB NAMES libopentelemetry_common.a HINTS "${OPENTELEMETRY_ROOT}/build/sdk/src/common" NO_DEFAULT_PATH)
-find_library(OPENTELEMETRY_TRACE_LIB NAMES libopentelemetry_trace.a HINTS "${OPENTELEMETRY_ROOT}/build/sdk/src/trace" NO_DEFAULT_PATH)
-find_library(OPENTELEMETRY_EXPORTER_LIB NAMES libopentelemetry_exporter_ostream_span.a HINTS "${OPENTELEMETRY_ROOT}/build/exporters/ostream" NO_DEFAULT_PATH)
-find_library(OPENTELEMETRY_RESOURCE_LIB NAMES libopentelemetry_resources.a HINTS "${OPENTELEMETRY_ROOT}/build/sdk/src/resource" NO_DEFAULT_PATH)
-
-if(OPENTELEMETRY_COMMON_LIB AND OPENTELEMETRY_TRACE_LIB AND OPENTELEMETRY_EXPORTER_LIB AND OPENTELEMETRY_RESOURCE_LIB)
-  message(STATUS "Found opentelemetry libraries")
-else()
-  message(SEND_ERROR "Did not find opentelemetry libraries")
-endif()
+# Use find_package to include OpenTelemetry C++
+find_package(opentelemetry-cpp CONFIG REQUIRED)
```

Comment:
This assumes that one runs `cmake install` above to install the opentelemetry-cpp package globally which is not the case from the instructions above.

I tried to get this working without running `cmake install` and pointing find package to the `build/` folder similar how we do it for oatpp, such that an install is not required, but that didn't work and threw me some errors. So please do the following:

Add a wording above that people run `cmake --install . --prefix ../../otel-cpp` above, this will install OpenTelemetry C++ in another subfolder contained in the `otel-cpp-starter`, afterwards the following is required for the find_package:

```cmake
find_package(opentelemetry-cpp CONFIG REQUIRED)

# Link against each OpenTelemetry C++ library
target_link_libraries(dice-server PRIVATE
                      ${OATPP_LIB}
                      ${OPENTELEMETRY_CPP_LIBRARIES})
```

Reply:
Will do! Thank you for catching that.